elmslie-samples/compose-paging/src/main/java/vivid/money/elmslie/samples/android/compose/view/PagingFragment.kt

```diff
@@ -41,8 +37,10 @@ class PagingFragment : ElmComponentFragment<PagingEvent, PagingEffect, PagingSta
 //        )
     }
 
-    override fun handleEffect(effect: PagingEffect) = when (effect) {
-        is PagingEffect.LoadError -> Unit // Not ui related effect will be handled here
-        is PagingEffect.RefreshError -> Unit
-    }
+    override fun createStore(stateHandle: SavedStateHandle) = pagingStoreFactory()
+
+//    override fun handleEffect(effect: PagingEffect) = when (effect) {
```

Comment:
remove?

Reply:
PagingFragment was broken in the previous MR. Let's fix it at once in separate MR